package/gluon-client-bridge/luasrc/lib/gluon/upgrade/320-gluon-client-bridge-wireless

```diff
@@ -42,6 +38,73 @@ util.foreach_radio(uci, function(radio, index, config)
 		ifname = suffix and 'client' .. suffix,
 		disabled = disabled or false,
 	})
+end
+
+local function configure_owe(radio, index, config, radio_name)
+	local name_client = 'client_' .. radio_name
+	local name_owe = 'owe_' .. radio_name
+	local suffix = radio_name:match('^radio(%d+)$')
```

Comment:
radio_id?

Reply:
i sued to change to the old name (which is also fine IMHO)